### DISCUSSION TITLE
Question about logging and qDebug()

### DISCUSSION THREAD

**Discussion by githubdoe** [githubdoe/DFTFringe#170] [Announcements]
Here is an example of how qDebug text is shown when I'm debugging code in Qt IDE.  It has a header that just gets in my way.  Is there anyway I can prevent that?

[2024-11-13 15:45:55.123] [info] DFTFringe Dale7.3.3 started
[2024-11-13 15:45:55.139] [debug] QT message handler: init center 4
[2024-11-13 15:45:55.180] [debug] QT message handler:  main2

**Comment by gr5** [comment]
You mean the date and time?  Or do you mean the first line?

**Comment by githubdoe** [comment]
I call the date,time,type "QT message handler:"  the header of the line.  I hope they can be turned off.  There is more the than the actual useful debug info.

**Comment by githubdoe** [comment]
Never mind I think I found the code in main that I can ifdef out for me.

**Comment by atsju** [comment]
Try adding `spdlog::set_pattern("%v");`

Documentation is here https://github.com/gabime/spdlog/wiki/3.-Custom-formatting.

The advantage of doing this I see is that you will continue to run spdlog and if something breaks in logging process you will notice.
